src/features/TrendingMovies/components/ImageSlider.tsx

```diff
@@ -0,0 +1,70 @@
+import { useState } from 'react';
```

Comment:
You called folder 'components' but at all other places you use name 'ui' Please stick with one naming everywhere, you can rename this folder with 'ui' or rename all other places with 'components' up to you, but be constant